MAUI/Expander/appearance.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Appearance in .NET MAUI Expander control | Syncfusion
+description: Learn here all about Appearance support in Syncfusion .NET MAUI Expander (SfExpander) control and more.
+platform: MAUI
+control: SfExpander
+documentation: ug
+---
+
+# Appearance in .NET MAUI Expander (SfExpander)
+
+The Expander allows to customize the appearance of the Icon and provides various functionalities to the users.
```

Comment:
The .NET MAUI SfExpander comes with built-in support for customizing the appearance of the header.